Angular/src/app/app.component.ts

```diff
@@ -1,10 +1,29 @@
-import { Component } from '@angular/core';
+import {Component, OnInit} from '@angular/core';
+
+import { LoginService } from './services/login.service';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
-export class AppComponent {
-  title = 'Angular 2019 -> ';
+export class AppComponent implements OnInit {
+    title = 'Angular 2019 -> ';
+
+    public isUserLogin: boolean = false;
+
+    constructor(private loginService: LoginService) {}
+
+    ngOnInit(): void {
+      this.getUserStatus();
+    }
+
+    getUserStatus() {
```

Comment:
Код можно упростить и удалить ненужный метод `getUserLoginStatus`. Так же, `getUserStatus` - некорректно, т.к. функция не соответствует действию - ничего не возвращает и присваивает переменной `isUserLogin` значение. Более релевантным будет название `setUserLoginStatus`.

```suggestion
   ngOnInit(): void {
      this.setUserLoginStatus()();
    }

    setUserLoginStatus(): void {
      const user = this.loginService.getUserInfo();
      this.isUserLogin = user.isAuthenticated;
    }
```